x-pack/plugins/security_solution/public/management/components/artifact_list_page/artifact_list_page.tsx

```diff
@@ -163,6 +163,7 @@ export const ArtifactListPage = memo<ArtifactListPageProps>(
     });
 
     const policiesRequest = useGetEndpointSpecificPolicies({
+      perPage: 1000,
```

Comment:
just one suggestion: add a test (unit) to this page component thta validates that we call the API with a `perPage` value of `1000`